docs/run_local_watcher.md

```diff
@@ -14,7 +14,7 @@ The `docker-compose` tooling in the root of `elixir-omg` allows users to run the
 
 2) From the root of the `elixir-omg` execute:
 
-- `docker-compose -f docker-compose.yml -f docker-compose-watcher-mac.yml up` (Mac)
-- `docker-compose -f docker-compose.yml -f docker-compose-watcher-non-mac.yml up` (Linux/Windows)
+- `docker-compose -f docker-compose.yml -f docker-compose-watcher-mac.yml up watcher postgres` (Mac)
+- `docker-compose -f docker-compose.yml -f docker-compose-watcher-non-mac.yml up watcher postgres` (Linux/Windows)
```

Comment:
Is this just so that we don't load `elixir-omg` service?

Reply:
yeah!